### ISSUE TITLE
The app turns on devices and leave them on

### ISSUE THREAD

**Issue by glockenbach** [yhaugom/com.telldus#38]
First, thank you for a great app!

I have noticed that the app for some reason turns on devices but not turning them back off. 
Is it really necessary to turn to turn them on at all?

See screen shot:
![image](https://user-images.githubusercontent.com/60673070/134512454-5b035e62-6a37-4d8e-bf8a-afdbbe2f3367.png)

**Comment by yhaugom** [comment]
I am not sure what this is. I do not turn anything on or off in my code. All devices are pretty standard Z-wave devices, and follow Athoms guidelines. For which plug-in switch is this? Mini or the larger one?

**Comment by glockenbach** [comment]
This is a mini-plug. I have the larger one too and can observe if it will happen also there. 

I should say this happens very rarely. I can imagine it is nothing caused by the app but it is a strange behaviour. The use case is obvious. No one wants plugs randomly turned on, regardless if it is a heater or not. I will see if I can file a ticket to Athom too and link it here.

**Comment by yhaugom** [comment]
Is it possible at all, that it happens when someone pushes the small button on the side of the plug? I think that will be displayed in the timeline as you show above. I do not have a working plug at hand myself, since it stopped working a few months ago.

**Comment by glockenbach** [comment]
I will confirm this on the weekend. It may also be true and somehow bring some light. 
However, the house was empty at this particular time. I have observed the behaviour and therfore made a "listener notification" for all heaters. 
In parallel I have posted an issue to Athom.

**Comment by glockenbach** [comment]
What you are suggesting is correct. The same event is triggered when I manually push the button. 

The diagnostics report which I sent to Athom was too old to cover. When this happens again I will post a new ticket to Athom and if I find a resolution to the problem I will post it here.
